Holyhead-Harpies/api-sprint-1-ml
# Testing

## Status
**READY**

## Migrations
NO

## Description
Tests for Model validations

## Todos


## Steps to Test or Reproduce
```
git fetch origin testing
git checkout testing
bundle
rails db:migrate
rails db:seed
```

All tests should pass, except Training Programs.

## Impacted Areas in Application
Added Rspec Gem, files and directories
